### PR TITLE
Paillier division

### PR DESCRIPTION
**Pull request by iamtrask**
This fixes the issue with mine.js that caused the following error found by @anoff :

mine_1     | Traceback (most recent call last):
mine_1     |   File "/usr/bin/syft_cmd", line 4, in <module>
mine_1     |     __import__('pkg_resources').run_script('syft==0.1.0', 'syft_cmd')
mine_1     |   File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 743, in run_script
mine_1     |     self.require(requires)[0].run_script(script_name, ns)
mine_1     |   File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 1505, in run_script
mine_1     |     exec(script_code, namespace, namespace)
mine_1     |   File "/usr/lib/python3.6/site-packages/syft-0.1.0-py3.6.egg/EGG-INFO/scripts/syft_cmd", line 141, in <module>
mine_1     |   File "/usr/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
mine_1     |     out=out, **kwargs)
mine_1     |   File "/usr/lib/python3.6/site-packages/numpy/core/_methods.py", line 73, in _mean
mine_1     |     ret, rcount, out=ret, casting='unsafe', subok=False)
mine_1     | TypeError: unsupported operand type(s) for /: 'PaillierTensor' and 'int'

### CODE REVIEW

File: syft/he/paillier/basic.py [hunk 1]
```diff
@@ -65,6 +65,19 @@ def __mul__(self, tensor):
         else:
             return PaillierTensor(self.public_key, self.data * float(tensor), False)
 
+    def __truediv__(self, tensor):
+        """Performs element-wise addition between two tensors"""
```
Comment (anoff) [inline syft/he/paillier/basic.py]: Isn't it `division` instead of `addition` in this case?

File: tests/test_paillier.py [hunk 2]
```diff
@@ -122,3 +122,33 @@ def testScalar(self):
...
+        p, s = KeyPair().generate()
+
+        x = PaillierTensor(p, np.array([3., 8., 15., 24., 35.]))
+        x2 = TensorBase(np.array([3, 4, 5, 6, 7.]))
+
+        y = x / x2
+        print(y.decrypt(s))
+        self.assertTrue(y.decrypt(s) == np.array([1., 2., 3., 4., 5.]))
+
+    def testInline(self):
```
Comment (anoff) [inline tests/test_paillier.py]: isn't that identical to `testBasic` except how the results are stored?

Comment (iamtrask) [reply tests/test_paillier.py]: yeah... for some functions its made a difference though. in this case i didn't create a separate "inline" function.

Comment (iamtrask) [reply tests/test_paillier.py]: i just took the relevant ones for multiplication and changed them for addition

**Pull request merged**